### PR TITLE
VE-2054: Display notification at the top of the page

### PR DESCRIPTION
**Pull request by pauloslund**

### CODE REVIEW

File: extensions/wikia/EmergencyBroadcastSystem/EmergencyBroadcastSystemHooks.class.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+class EmergencyBroadcastSystemHooks extends WikiaController {
+	public function onBeforePageDisplay( OutputPage $out, $skin ) {
+		$app = F::app();
+		$assetsManager = AssetsManager::getInstance();
+
+		if ( $app->checkSkin( 'oasis' ) ) {
+			$scss = 'emergency_broadcast_system_scss';
+
+			foreach ( $assetsManager->getURL( $scss ) as $url ) {
+				$out->addStyle( $url );
```
Comment (inez) [inline extensions/wikia/EmergencyBroadcastSystem/EmergencyBroadcastSystemHooks.class.php:12]: There should be easier way to add CSS to the page. Also - consider adding your new CSS to some already existing group - so there will not be a need to additional HTTP request.

File: extensions/wikia/EmergencyBroadcastSystem/css/EmergencyBroadcastSystem.scss [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+		margin-bottom: $ebs-standard-spacing;
+	}
+
+	.ebs-unread {
+		background: $ebs-unread-background;
+		border-radius: 100%;
+		color: $ebs-unread-color;
+		font-size: $ebs-unread-font-size;
+		height: $ebs-unread-size;
+		left: 36px; // 36px is a magic number currently... Find a way to clean this up
```
Comment (inez) [inline extensions/wikia/EmergencyBroadcastSystem/css/EmergencyBroadcastSystem.scss]: Should this be cleanup before merging?

File: extensions/wikia/EmergencyBroadcastSystem/templates/EmergencyBroadcastSystem_index.php [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<section class="ebs-container">
+	<p class="ebs-unread"><?= $nonPortableCount ?></p>
+	<div class="ebs-content">
+		<h3>Infoboxes on this wiki are not properly rendered on mobile devices</h3>
```
Comment (inez) [inline extensions/wikia/EmergencyBroadcastSystem/templates/EmergencyBroadcastSystem_index.php:4]: i18n please

Comment (pauloslund) [inline extensions/wikia/EmergencyBroadcastSystem/templates/EmergencyBroadcastSystem_index.php:4]: Was planning to bring in the i18n in another ticket/PR (VE-2063)

File: extensions/wikia/EmergencyBroadcastSystem/EmergencyBroadcastSystem.php [hunk 4]
```diff
@@ -0,0 +1,14 @@
+<?php
+
+$wgExtensionCredits['other'][] = array(
+	'name' => 'EmergencyBroadcastSystem',
+	'version' => '1.0',
+	'author' => 'Paul Oslund',
```
Comment (inez) [inline extensions/wikia/EmergencyBroadcastSystem/EmergencyBroadcastSystem.php:6]: oh man! this should be an array!!! ;-)

**Pull request merged**